prompting/settings.py

```diff
@@ -136,6 +136,9 @@ def load_env(values):
         values["LLM_MAX_MODEL_LEN"] = int(os.environ.get("LLM_MAX_MODEL_LEN", 4096))
         values["NEURON_LLM_MAX_ALLOWED_MEMORY_IN_GB"] = os.environ.get("MAX_ALLOWED_VRAM_GB", 62)
         values["NEURON_GPUS"] = os.environ.get("NEURON_GPUS", 1)
+        if values["NEURON_GPUS"] != 1:
```

Comment:
Doesn't matter to much, but shouldn't it be
```suggestion
        if values["NEURON_GPUS"] > 1:
```